src/main/java/net/coderbot/iris/mixin/MixinGameRenderer.java

```diff
@@ -18,6 +25,14 @@
 public class MixinGameRenderer {
 	// TODO: This probably won't be compatible with mods that directly mess with the GL projection matrix.
 	// https://github.com/jellysquid3/sodium-fabric/blob/1df506fd39dac56bb410725c245e6e51208ec732/src/main/java/me/jellysquid/mods/sodium/client/render/chunk/shader/ChunkProgram.java#L56
+	@Inject(method = "<init>", at = @At("TAIL"))
```

Comment:
I wonder if it would be better to put this code in the entrypoint instead of in a mixin?

Reply:
I attempted to do so, however LWJGL gave an error last I checked due to GLFW not being initialized yet.